src/controllers/dynakube/dynakube_controller.go

```diff
@@ -316,7 +316,7 @@ func (controller *DynakubeController) reconcileActiveGate(ctx context.Context, d
 
 func (controller *DynakubeController) reconcileActiveGateProxySecret(ctx context.Context, dynakubeState *status.DynakubeState) bool {
 	gen := agproxysecret.NewActiveGateProxySecretGenerator(controller.client, controller.apiReader, dynakubeState.Instance.Namespace, log)
-	if dynakubeState.Instance.HasProxy() {
+	if !dynakubeState.Instance.FeatureActiveGateIgnoreProxy() && dynakubeState.Instance.HasProxy() {
```

Comment:
The property mentioned before can then be used here too.  See the comment for the activegate proxy flag